lib/routes/routes.dart

```diff
@@ -55,13 +50,8 @@ class SetupRoutes {
           contactSelectedHistory: args['showSelectedData'],
         );
       },
-      Routes.GROUPS: (context) {
-        Map<String, dynamic>? args =
-            ModalRoute.of(context)!.settings.arguments as Map<String, dynamic>?;
-        return GroupList();
-      },
+      Routes.GROUPS: (context) => GroupList(),
```

Comment:
Is group coming from `at_contacts_group_flutter` ?

Reply:
@sachins-geekyants It's unused anymore
I will remove it to prevent misunderstanding
Thanks!